huggingface_basics.py

```
@@ -0,0 +1,454 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# <a href="https://colab.research.google.com/github/olavvn/7-huggingface_practice/blob/master/huggingface_basics.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
+
+# In[ ]:
+
+
+get_ipython().system('pip install transformers datasets huggingface_hub')
```

Ruff (0.13.3)

Undefined name `get_ipython`

(F821)